compiler-core/src/language_server/tests/mod.rs

```diff
@@ -82,6 +82,21 @@ impl LanguageServerTestIO {
         path
     }
 
+    pub fn hex_dep_module(&self, dep: &str, name: &str, code: &str) -> Utf8PathBuf {
```

Comment:
Could you rename `dep_module` to `path_dep_module` to make it clear the difference please.